src/io/flutter/utils/FlutterModuleUtils.java

```diff
@@ -324,6 +329,22 @@ public static boolean isDeprecatedFlutterModuleType(@NotNull Module module) {
     return FlutterModuleUtils.declaresFlutter(module);
   }
 
+  public static boolean isInFlutterAndroidModule(@NotNull Project project, @NotNull VirtualFile file) {
```

Comment:
Is this impl. the same as the private method of the same name in `FlutterExternalIdeActionGroup.java`?